.github/workflows/appstream-check.yaml

```diff
@@ -0,0 +1,31 @@
+name: Validate appstream metadata xml
+on:
+  workflow_dispatch: # Manual trigger
+  pull_request:
+    paths:
+      - qt/res/app.organicmaps.desktop.metainfo.xml
+      - .github/workflows/appstream-check.yaml  # Run check on self change
+
+jobs:
+  validate-appstream:
+    name: Validate appstream metadata xml
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
```

Comment:
This is not needed for this job type.

Reply:
Right, we only have a single job, so nothing else can be marked as failed. Removed.